Felienne/hedy
# [FEATURE] Enables teacher to reset student password

**Description**
In this PR we implement a new feature, enabling teachers too reset a students password. As discussed earlier some children don't have access to a mailadres at the school, making it hard to invite them to classes and/or reset their passwords. With the implementation of #1764 the first one is fixed. In this PR we fix the other issue.

We implement an additional column on the class-overview page called `change_password` pressing it will pop-up the modal for the teacher to enter a new password. Due to the table getting crowded we also implement dynamic column hiding/showing as already implemented within the admin page in #1740.

**Fix for**
This PR fixes #1736.

**How to test**
Navigate to the class-overview and notice the new attribute selection too at the top of the page. Make sure the "change password" column is shown and verify that you can indeed change the password of a student.
